### ISSUE TITLE
math.bits.div_64 has incorrect behaviour

### ISSUE THREAD

**Issue by playXE** [vlang/v#13188]
Found while implementing math.uint module: https://github.com/vlang/v/pull/13186. 

Error when trying to convert uint128 to string vs go version:
```zsh
➜  v git:(math-uint) ✗ go run file.go
call div64(0,23,10000000000000000000)
0 0 23
23%                                                                                                                                                                                                                                                                               
➜  v git:(math-uint) ✗ ./v run file.v
call div_64(0, 23, 10000000000000000000)
0 42 4275113695319687191
call div_64(0, 42, 10000000000000000000)
0 77 4763251095801167914
call div_64(0, 77, 10000000000000000000)
0 142 399293675635474509
V panic: array.get: index out of range (i == -1, a.len == 40)
v hash: 9fd65b5
/tmp/v_1000/file.5310574360648299419.tmp.c:6031: at _v_panic: Backtrace
/tmp/v_1000/file.5310574360648299419.tmp.c:5562: by array_get
/tmp/v_1000/file.5310574360648299419.tmp.c:11379: by math__uint__Uint128_str
/tmp/v_1000/file.5310574360648299419.tmp.c:11423: by main__main
/tmp/v_1000/file.5310574360648299419.tmp.c:11791: by main
```

**Comment by spytheman** [comment]
fixed in V 2a3a4cf

**Comment by spytheman** [comment]
```v
import math.bits

println( bits.div_64(0, 23, 10000000000000000000) )
println( bits.div_64(0, 42, 10000000000000000000) )
println( bits.div_64(0, 77, 10000000000000000000) )
```
produces:
```
(0, 23)
(0, 42)
(0, 77)
```
